dll/win32/msafd/misc/dllmain.c

```diff
@@ -2549,6 +2549,22 @@ WSPIoctl(IN  SOCKET Handle,
             ((SOCKET_ADDRESS_LIST*)lpvOutBuffer)->Address[0].iSockaddrLength = sizeof(Socket->SharedData->WSLocalAddress);
             ((SOCKET_ADDRESS_LIST*)lpvOutBuffer)->Address[0].lpSockaddr = &Socket->SharedData->WSLocalAddress;
 
+            Errno = NO_ERROR;
+            Ret = NO_ERROR;
+            break;
+        case SIO_UDP_CONNRESET:
```

Comment:
I feel like this should go to some wsh* library (wshtcpip?), and not into "generic" msafd